src/Provider/AkismetProvider.php

```diff
@@ -9,24 +9,37 @@
 
 namespace Flarum\Akismet\Provider;
 
+use Flarum\Extension\ExtensionManager;
 use Flarum\Foundation\AbstractServiceProvider;
+use Flarum\Foundation\Application;
+use Flarum\Foundation\Config;
 use Flarum\Http\UrlGenerator;
 use Flarum\Settings\SettingsRepositoryInterface;
-use TijsVerkoyen\Akismet\Akismet;
+use Flarum\Akismet\Akismet;
 
 class AkismetProvider extends AbstractServiceProvider
 {
     public function register()
     {
-        $this->app->bind(Akismet::class, function () {
+        $this->container->bind(Akismet::class, function () {
```

Comment:
```suggestion
        $this->container->bind(Akismet::class, function (Container $container) {
```

We prefer this approach because it works better with Octane, should we eventually choose to switch to it or something like it.